polaris/polaris/sep24/deposit.py

```diff
@@ -256,6 +263,13 @@ def get_interactive_deposit(request: Request) -> Response:
     elif content is None:
         content = {}
 
+    if registered_scripts_func != calculate_fee:
```

Comment:
@JakeUrban Same issue here.
Maybe should be `if registered_scripts_func != scripts`?